content/manager_webui/default-widgets-ref.md

```diff
@@ -92,12 +97,12 @@ Displays a chart or charts (up to 5) presenting metric data for the current depl
 
 #### Configuration
 
-You must supply at least one metric or database query in the widget configuration. You also need to supply the deployment's ID, either in the page context, or by specifying it in the widget configuration. 
+You must supply at least one metric or database query in the widget configuration. You also need to supply the Deployment ID and Node instance ID, either in the page context, or by specifying it in the widget configuration. 
 
 The following list provides information regarding parameters that can be specified for this widget. 
 
 * `Refresh Time Interval` - how frequently the data in the widget is refreshed (in secs).
-* `Node filter` - the node instance for which you want to display data. Deployment ID and Node Instance ID must be set in the configuration or as part of the page's context. You can set deployment ID and node instance ID in page's context using [Resource Filter](#resource-filter)
+* `Node filter` - the node instance for which you want to display data. Deployment ID and Node Instance ID must be set in the configuration or as part of the page's context. You can set Deployment ID and Node instance ID in page's context using [Resource Filter](#resource-filter)
```

Comment:
the node instance for which you want to display data. > Show only data for the specified nodes.
page's context > page context
using `[Resource Filter](#resource-filter)` > with the `[Resource Filter](#resource-filter)`.

Reply:
Updated.